app/src/main/java/com/ferranpons/issposition/MainActivity.kt

```diff
@@ -0,0 +1,292 @@
+import com.ferranpons.issposition.peopleInSpace.PeopleAdapter
+import com.google.android.gms.common.ConnectionResult
+import com.google.android.gms.common.GooglePlayServicesUtil
+import com.google.android.gms.maps.CameraUpdateFactory
+import com.google.android.gms.maps.GoogleMap
+import com.google.android.gms.maps.SupportMapFragment
+import com.google.android.gms.maps.model.BitmapDescriptorFactory
+import com.google.android.gms.maps.model.CameraPosition
+import com.google.android.gms.maps.model.LatLng
+import com.google.android.gms.maps.model.MarkerOptions
+
+class MainActivity : AppCompatActivity(), IssTrackingViewInterface {
+
+    private val map: GoogleMap? = null
+    private var issTrackingPresenter: IssTrackingPresenterInterface? = null
+
+    @BindView(R.id.peopleInSpaceListView) var peopleInSpaceListView: ListView? = null
+
+    @BindView(R.id.progressPeopleInSpace) var peopleInSpaceProgressBar: ProgressBar? = null
+
+    @BindView(R.id.collapsePeople) var peopleInSpaceCollapseImage: ImageView? = null
+
+    @BindView(R.id.retryPeople) var peopleInSpaceRetryImage: ImageView? = null
+
+    @BindView(R.id.collapseLayout) var peopleInSpaceCollapseButton: LinearLayout? = null
+
+    @BindView(R.id.passTimesListView) var passTimesListView: ListView? = null
+
+    @BindView(R.id.progressPassTimes) var passTimesProgressBar: ProgressBar? = null
+
+    @BindView(R.id.collapsePassTimesLayout) var passTimesCollapseButton: LinearLayout? = null
+
+    @BindView(R.id.collapsePassTimes) var passTimesCollapseImage: ImageView? = null
+
+    @BindView(R.id.retryPassTimes) var passTimesRetryImage: ImageView? = null
+
+    private var connectivityChange: ConnectivityChange? = null
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+        ButterKnife.bind(this)
+        //flipPeopleInSpaceCollapseButton()
+        //flipPassTimesCollapseButton()
+        issTrackingPresenter = IssTrackingPresenter(IssTrackingInteractor(IssTrackingApi.getIssTrackingApi("http://api.open-notify.org")))
+        (issTrackingPresenter as IssTrackingPresenter).setView(this)
+    }
+
+    @OnClick(R.id.collapseLayout)
+    fun collapsePeopleInSpaceView() {
+        if (peopleInSpaceListView?.visibility == View.VISIBLE) {
+            peopleInSpaceListView?.visibility = View.GONE
+            //flipPeopleInSpaceCollapseButton()
+        } else {
+            peopleInSpaceListView?.visibility = View.VISIBLE
+            //flipPeopleInSpaceCollapseButton()
+        }
+    }
+
+    @OnClick(R.id.collapsePassTimesLayout)
+    fun collapsePassTimesView() {
+        if (passTimesListView?.visibility == View.VISIBLE) {
+            passTimesListView?.visibility = View.GONE
+            //flipPassTimesCollapseButton()
+        } else {
+            passTimesListView?.visibility = View.VISIBLE
+            //flipPassTimesCollapseButton()
+        }
```

Comment:
IFs have a return in Kotlin, so you could say: paasTimesListView.visibility =  if ...